src/index.html

```diff
@@ -7,8 +7,31 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <a href="#home" class="logo"><img src="/images/logo.png" alt="logo"></a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__link"><a class="link is-active" href="#">Apple</a></li>
+          <li class="nav__link"><a class="link" href="#">Samsung</a></li>
+          <li class="nav__link"><a class="link" href="#">Smartphones</a></li>
+          <li class="nav__link"><a class="link computer__link" data-qa="hover" href="#">Laptops & computers</a></li>
```

Comment:
```suggestion
          <li class="nav__link">
            <a class="link" href="#">Smartphones</a>
          </li>
          
          <li class="nav__link">
            <a 
              class="link computer__link" 
              data-qa="hover" 
              href="#"
            >Laptops & computers</a>
          </li>
```
make empty line between siblings.
new elevent write on new line, if tag have three or more attributes - place each on new line
check code style in other places in html